src/index.js

```diff
@@ -0,0 +1,94 @@
+temp = 70
+
+const updateTemp = temp => {
+    const tempFarenheitContainer= document.getElementById("tempFarenheit");
+    tempFarenheitContainer.textContent = temp;
+}
```

Comment:
When we update the temperature, we also want the color and the ground to change, so this would be a great place to call those helpers as well.

```js
    updateGround(temp);
    updateTempColor(temp);
```